framework/core/allocated.h

```diff
@@ -80,8 +98,32 @@ void init(const DeviceType &device)
 	init(allocator_info);
 }
 
+/**
+ * @brief Shuts down the VMA allocator and releases all resources.  Idempotent, but should be paired with `init` and attempting to run examples
```

Comment:
Ahm... What do you mean here with "and attempting to run examples"?

Reply:
"and" was supposed to be "when".  Will clean up